# References #

  * Abdo, Z., V. N. Minin, P. Joyce, and J. Sullivan. 2005. Accounting for uncertainty in the tree topology has little effect on the decision-theoretic approach to model selection in phylogeny estimation. Molecular Biology and Evolution 22:691-703.
  * Akaike, H. 1974. A new look at the statistical model identification. IEEE Transactions on Automatic Control 19:716-723.
  * Alfaro, M. E., and J. P. Huelsenbeck. 2006. Comparative performance of Bayesian and AIC-based measures of phylogenetic model uncertainty. Systematic Biology 55:89-96.
  * Burnham, K. P., and D. R. Anderson. 2003. Model selection and multimodel inference: a practical information-theoretic approach. Springer-Verlag, New York, NY.
  * Burnham, K. P., and D. R. Anderson. 1998. Model selection and inference: a practical information-theoretic approach. Springer-Verlag, New York, NY.
  * Felsenstein, J. 1988. Phylogenies from molecular sequences: inference and reliability. Annual Review of Genetics 22:521-565.
  * Felsenstein, J. 1981. Evolutionary trees from DNA sequences: A maximum likelihood approach. Journal of Molecular Evolution 17:368-376.
  * Felsenstein, J. 2005. PHYLIP (Phylogeny Inference Package). Department of Genome Sciences, University of Washington, Seattle.
  * Gascuel, O. 1997. BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data. Molecular Biology and Evolution 14:685-695.
  * Gilbert, D. 2007. ReadSeq. Indiana University, Bloomington.
  * Goldman, N. 1993a. Simple diagnostic statistical test of models of DNA substitution. Journal of Molecular Evolution 37:650-661.
  * Goldman, N. 1993b. Statistical tests of models of DNA substitution. Journal of Molecular Evolution 36:182-198.
  * Goldman, N., and S. Whelan. 2000. Statistical tests of gamma-distributed rate heterogeneity in models of sequence evolution in phylogenetics. Molecular Biology and Evolution 17:975-978.
  * Guindon, S., and O. Gascuel. 2003. A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Systematic Biology 52:696-704.
  * Hasegawa, M., K. Kishino, and T. Yano. 1985. Dating the human-ape splitting by a molecular clock of mitochondrial DNA. Journal of Molecular Evolution 22:160-174.
  * Hoeting, J. A., D. Madigan, and A. E. Raftery. 1999. Bayesian model averaging: A tutorial. Statistical Science 14:382-417.
  * Huelsenbeck, J. P., B. Larget, and M. E. Alfaro. 2004. Bayesian Phylogenetic Model Selection Using Reversible Jump Markov Chain Monte Carlo. Molecular Biology and Evolution 21:1123-1133.
  * Hurvich, C. M., and C.-L. Tsai. 1989. Regression and time series model selection in small samples. Biometrika 76:297-307.
  * Johnson, J. B., and K. S. Omland. 2003. Model selection in ecology and evolution. Trends in Ecology & Evolution 19:101-108.
  * Jukes, T. H., and C. R. Cantor. 1969. Evolution of protein molecules. Pp. 21-132 in H. M. Munro, ed. Mammalian Protein Metabolism. Academic Press, New York, NY.
  * Kendall, M., and A. Stuart. 1979. The Advanced Theory of Statistics. Charles Griffin, London.
  * Kimura, M. 1981. Estimation of evolutionary distances between homologous nucleotide sequences. Proceedings of the National Academy of Sciences, U.S.A. 78:454-458.
  * Kimura, M. 1980. A simple method for estimating evolutionary rate of base substitutions through comparative studies of nucleotide sequences. Journal of Molecular Evolution 16:111-120.
  * Kullback, S., and R. A. Leibler. 1951. On information and sufficiency. Annals of Mathematical Statistics 22:79-86.
  * Madigan, D. M., and A. E. Raftery. 1994. Model selection and accounting for model uncertainty in graphical models using Occam's Window. Journal of the American Statistical Association 89:1335-1346.
  * Minin, V., Z. Abdo, P. Joyce, and J. Sullivan. 2003. Performance-based selection of likelihood models for phylogeny estimation. Systematic Biology 52:674-683.
  * Ohta, T. 1992. Theoretical study of near neutrality. II. Effect of subdivided population structure with local extinction and recolonization. Genetics:917-923.
  * Pol, D. 2004. Empirical problems of the hierarchical likelihood ratio test for model selection. Systematic Biology 53:949-962.
  * Posada, D. 2001. The effect of branch length variation on the selection of models of molecular evolution. Journal of Molecular Evolution 52:434-444.
  * Posada, D. 2003. Using Modeltest and PAUP to select a model of nucleotide substitution. Pp. 6.5.1-6.5.14 in A. D.
  * Posada, D., and T. R. Buckley. 2004a. Model selection and model averaging in phylogenetics: advantages of the AIC and Bayesian approaches over likelihood ratio tests. Systematic Biology 53:793-808.
  * Posada, D., and T. R. Buckley. 2004b. Model selection and model averaging in phylogenetics: advantages of Akaike Information Criterion and Bayesian approaches over likelihood ratio tests. Systematic Biology 53:793-808.
  * Posada, D., and K. A. Crandall. 2001a. Selecting the best-fit model of nucleotide substitution. Systematic Biology 50:580-601.
  * Posada, D., and K. A. Crandall. 2001b. Selecting models of nucleotide substitution: an application to human immunodeficiency virus 1 (HIV-1). Molecular Biology and Evolution 18:897-906.
  * Raftery, A. E. 1996. Hypothesis testing and model selection. Pp. 163-187 in W. R. Gilks, S. Richardson, and D. J. Spiegelhalter, eds. Markov chain Monte Carlo in practice. Chapman & Hall, London ; New York.
  * Schwarz, G. 1978. Estimating the dimension of a model. The Annals of Statistics 6:461-464. Sugiura, N. 1978. Further analysis of the data by Akaike's information criterion and the finite corrections. Communications in Statistics–Theory and Methods A7:13-26.
  * Sullivan, J., and P. Joyce. 2005. Model selection in phylogenetics. Annual Review of Ecology, Evolution and Systematics 36:445-466.
  * Tamura, K., and M. Nei. 1993. Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees. Molecular Biology and Evolution 10:512-526.
  * Tavaré, S. 1986. Some probabilistic and statistical problems in the analysis of DNA sequences. Pp. 57-86 in R. M. Miura, ed. Some mathematical questions in biology - DNA sequence analysis. Amer. Math. Soc., Providence, RI.
  * Wasserman, L. 2000. Bayesian Model Selection and Model Averaging. Journal of Mathematical Psychology 44:92-107.
  * Whelan, S., and N. Goldman. 1999. Distributions of statistics used for the comparison of models of sequence evolution in phylogenetics. Molecular Biology and Evolution 16:1292-1299.
  * Yang, Z., N. Goldman, and A. Friday. 1995. Maximum likelihood trees from DNA sequences: a peculiar statistical estimation problem. Systematic Biology 44:384-399.
  * Zharkikh, A. 1994. Estimation of evolutionary distances between nucleotide sequences. Journal of Molecular Evolution 39:315-329.